lms/envs/common.py

```diff
@@ -1296,7 +1296,6 @@
 ]
 
 staff_grading_js = sorted(rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/staff_grading/**/*.js'))
-open_ended_js = sorted(rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/open_ended/**/*.js'))
```

Comment:
@stvstnfrd just a note, when you remove coffeescript in the future be sure to also remove the compilation rules.